### PR TITLE
Fix: Check status api to verify if airflow is setup

### LINKED ISSUE #4899: UI: Check `status` api to verify if airflow is setup
Check status api to notify user about airflow setup before allowing them to create ingestion pipelines

### PR DESCRIPTION
**Pull request by darth-coder00**
Closes #4899 
### Describe your changes :

I worked on the API integration and UI for airflow status API

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p>
When Airflow is up and running while creating service/ingestion, it checks silently for status
Screenshot 2022-05-13 at 12 26 26 PM

When Airflow is down on creating new service/ingestion, it checks silently for status
Screenshot 2022-05-13 at 11 35 08 AM

Now if needed, user can bring up the Airflow instance, click `check status` and resume ingestion once status is OK
Screenshot 2022-05-13 at 1 03 05 PM

If Airflow is down when User visits Ingestion tab on service details page
Screenshot 2022-05-12 at 11 33 14 PM

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolderIngestion.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  {
+    step: 1,
+    title: 'Custom Airflow Installation',
+    description:
+      'How to install OpenMetadata ingestion modules to an existing Airflow host.',
+    link: 'https://docs.open-metadata.org/integrations/airflow/custom-airflow-installation',
+  },
+  {
+    step: 2,
+    title: 'Airflow Lineage',
```
Comment (harshach) [inline openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolderIngestion.tsx:28]: remove this step 2, this is not related to ingestion

File: openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolderIngestion.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+ */
+
+import { uniqueId } from 'lodash';
+import { observer } from 'mobx-react';
+import React from 'react';
+
+const stepsData = [
+  {
+    step: 1,
+    title: 'Custom Airflow Installation',
```
Comment (harshach) [inline openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolderIngestion.tsx:21]: remove "Custom Airflow Installation" -> "Managed Airflow APIs"
OpenMetadata uses Airflow to run Ingestion Connectors. We developed Managed APIs to deploy ingestion connectors. Please refer to the guide here to use the OpenMetadata Airflow docker container or install the managed APIs in your Airflow installation.

Comment (harshach) [reply openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolderIngestion.tsx:21]: There should just be one Para, not multiple steps.
Show the error "Failed to find OpenMetadata - Managed Airflow APIs". below that add the text I wrote in first comment and link to the custom-airflow-installation

Comment (darth-coder00) [reply openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolderIngestion.tsx]: Done

File: openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolderIngestion.tsx (hunk 1 above)
Comment (darth-coder00) [reply openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolderIngestion.tsx]: Done

**Review by ShaileshParmar11 — APPROVED** [review]

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by vivekratnavel — APPROVED** [review]

**Pull request merged**